htdocs/product/stock/movement_list.php

```diff
@@ -789,16 +792,17 @@
 		print '<input class="flat maxwidth25" type="text" name="search_ref" value="'.dol_escape_htmltag($search_ref).'">';
 		print '</td>';
 	}
-	if (!empty($arrayfields['m.datem']['checked']))
+	if (! empty($arrayfields['m.datem']['checked']))
 	{
-		// Date
-		print '<td class="liste_titre nowraponall">';
-		print '<input class="flat" type="text" size="2" maxlength="2" placeholder="'.dol_escape_htmltag($langs->trans("Month")).'" name="month" value="'.$month.'">';
-		if (empty($conf->productbatch->enabled)) print '&nbsp;';
-		//else print '<br>';
-		$syear = $year ? $year : -1;
-		print '<input class="flat maxwidth50" type="text" maxlength="4" placeholder="'.dol_escape_htmltag($langs->trans("Year")).'" name="year" value="'.($syear > 0 ? $syear : '').'">';
-		//print $formother->selectyear($syear,'year',1, 20, 5);
+		print '<td class="liste_titre center">';
+		print '<div class="nowrap">';
+		print $langs->trans('From') . ' ';
+		print $form->selectDate($search_date_start?$search_date_start:-1, 'search_date_start', 0, 0, 1);
```

Comment:
Because date is an input of user the last parameter of selectDate here must be 'tzuserrel'

Reply:
I add 'tzuserrel'